### DISCUSSION TITLE
💥 Deprecation of the Dart Code Metrics integration

### DISCUSSION THREAD

**Discussion by helinanever** [codemagic-ci-cd/codemagic-docs#1767] [Announcements]
As the maintainers of the [dart_code_metrics](https://pub.dev/packages/dart_code_metrics) plugin are deprecating the free version of the plugin from July 16, 2023, we are going to remove the Dart Code Metrics integration utilizing this plugin. Read the plugin maintainers’ official announcement [here](https://dcm.dev/blog/2023/06/06/announcing-dcm-free-version-sunset/).

Your Flutter workflow editor workflows will be automatically migrated to remove the Dart Code Metrics analysis step from the workflow. If you are using the dart_code_metrics plugin with codemagic.yaml, please review your workflows in time to avoid build failures. For more information about using the paid version of Dart Code Metrics on CI/CD, please refer to [DCM docs](https://dcm.dev/docs/getting-started/#running-executable-on-cicd).

**Comment by helinanever** [comment]
The Dart Code Metrics integration has been removed from Flutter Workflow Editor.

**Reply by thomtomdup** [reply]
Hello.
Is it possible to re-add it for people who have a paid license ?
Currently the process added recently don’t give the same result from process than before. This was better.
Thank you.
